### DISCUSSION TITLE
Minimal laptop setup for plotting and farming

### DISCUSSION THREAD

**Discussion by Cl2pp** [Chia-Network/chia-blockchain#1380] [Q&A] [answered]
Hi, 

I was wondering if I could get a plot up and running with a minimal build (MacBook Pro 2016 / 2.2Ghz i7 / 16GB DDR3 / 250GB + 512GB USB3 Stick (<130mbit read/write)). As far as I understand the documentation it would be able to host K32. 

Long-term I am going to build a proper rig. This would be just for exploring farming with my current, very limited setup before upgrading. Thanks for any help in advance.

Best
Clem

**Comment by dorkmo** [comment, marked as answer]
it might be slow, but it should work

**Reply by Cl2pp** [reply]
Cool, thanks for the info!
